kolibri/core/auth/utils.py

```diff
@@ -10,3 +13,41 @@ def confirm_or_exit(message):
     if answer != "yes":
         print("Canceled! Exiting without touching the database.")
         sys.exit(1)
+
+
+def get_facility(facility_id=None, noninteractive=False):
```

Comment:
Given it's specifically a helper function for management commands (`noninteractive`, `CommandError`, etc), perhaps it would belong better in something like `kolibri/core/auth/management/commands/utils.py`? Or if it's only used by the one management command, for now it could just live inside the command file directly?

Reply:
I thought it awkward to live there because it can then be invoked with `manage utils`, even though it would fail.
But if its undocumented, I suppose it could live there?

Reply:
Discussed in person: put in management/utils.py